tumblr-gif-search/app.py

```diff
@@ -59,18 +62,12 @@ def index_restful(num_docs):
             read_mode='r',
         )
         data_json = {'data': [Document(text=text).dict() for text in input_docs]}
-        print(f'#### {len(data_json["data"])}')
+        f.logger.info(f'#### {len(data_json["data"])}')
```

Comment:
I dont think this message is really meaningful like this, but that is just my opinion here and it was there before.

Reply:
Yes, looks more like a debugging message. I removed it in the two examples https://github.com/jina-ai/examples/pull/460/commits/7cdd821aedb3a627ea5a15a00bc1308daa7b418e